Makefile

```diff
@@ -4,7 +4,7 @@ SHELL:=/bin/bash
 bash:
 	@if [[ ! -f /.dockerenv ]] ; then
 		echo 'Connecting to php_scaffolding docker instance...'
-		docker exec -ti -u dev php_scaffolding bash
+		docker exec -ti -u dev php-scaffolding bash
```

Comment:
Nope. 
* `php_scaffolding` is for the docker `container_name` (defined in the `docker-compose.yml`)
* `php-scaffolding` is the WORKDIR (defined in the `dockerfile: devops/dev/php.dockerfile`)

Reply:
You are right.
Thank you for pointing it out! :)